<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 245, 238);</string>
  </property>
  <widget class="QLabel" name="creditSumLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; </string>
   </property>
   <property name="text">
    <string>Credit sum</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="creditTermLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; </string>
   </property>
   <property name="text">
    <string>Credit term</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="termTypeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
</string>
   </property>
   <property name="text">
    <string>Term type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="percentLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; </string>
   </property>
   <property name="text">
    <string>Percent</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paymentTypeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
</string>
   </property>
   <property name="text">
    <string>Monthly payment type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="yearsButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
   </property>
   <property name="text">
    <string>years</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="monthsButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
   </property>
   <property name="text">
    <string>months</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="annuityButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
   </property>
   <property name="text">
    <string>annuity</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="differentButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
   </property>
   <property name="text">
    <string>different</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="creditSumLine">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 30px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="creditTermLine">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 30px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="percentLine">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 30px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mounthlyPaymentLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
</string>
   </property>
   <property name="text">
    <string>Monthly payment </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="overPaymentLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
</string>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="FullPaymentLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
</string>
   </property>
   <property name="text">
    <string>Full payment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MounthlyPaymentLabelResult">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="overPaymentLabelResult">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="FullPaymentLabelResult">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculateButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>701</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
